### DISCUSSION TITLE
关于IC2中尝试引爆<核弹>而导致的游戏崩溃问题

### DISCUSSION THREAD

**Discussion by WaterDouble** [IndustrialCraftMC/IndustrialCraft2#25] [【漏洞反馈】Bugs Report]
64个<铀-238> 64个<工业TNT>：
[错误报告-2023-5-7_0.29.12.zip](https://github.com/AbbottMc/IndustrialCraft2/files/11412839/-2023-5-7_0.29.12.zip)

1个<铀-238> 1个<工业TNT>：
[错误报告-2023-5-7_0.45.08.zip](https://github.com/AbbottMc/IndustrialCraft2/files/11412843/-2023-5-7_0.45.08.zip)

以上为<核弹>内放置特殊数值的必要素材时，尝试点燃<核弹>而游戏崩溃后各自输出的<错误报告>。

游戏版本：使用1.19.2 forge-43.2.0 Op-I1-pre4

问题描述：核弹中填充任意数量的<铀-238>和任意数量的<工业TNT> 使用<打火石>尝试点燃时游戏无响应后游戏崩溃，重新进入游戏后发现<核弹>的爆炸效果并未发动，核弹原地消失。

**Comment by liuhaoyu-China** [comment]
同样的现象，核弹的威力还没有普通TNT大，还会崩溃，应该不是简单的电脑带不动
